### PR TITLE
FPO-309: Adds commodities data source

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

FPO-309

### What?

I have added/removed/altered:

- [x] Adds commodities data source

### Why?

I am doing this because:

- This used to be included statically but got removed after a refactor
- This is needed by Gurdip for her work on the hierarchical models

### AC

- Model passes benchmarks

### CODE REVIEW

File: data_sources/commodities.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        creates_codes: bool = False,
+        multiplier: int = 1,
+    ):
+        url = f"{self.HOST}{self.PATH}"
+        today = datetime.datetime.now()
+        year = str(today.year)
+        month = str(today.month).rjust(2, "0")
+        day = str(today.day).rjust(2, "0")
+
+        url = (
```
Comment (alexdesi) [inline data_sources/commodities.py:31]: I don't like reassigning the variable, especially with a different type, 
but it's just personal, nothing wrong :smile_cat:

Comment (willfish) [reply data_sources/commodities.py:31]: Its the same type, though

Comment (willfish) [reply data_sources/commodities.py:31]: We could create a new concept (unsubbed_url) or something

**Review by alexdesi — APPROVED** [review]
:white_check_mark:

**Pull request merged**